lib/modules/manager/cargo/readme.md

```diff
@@ -4,3 +4,20 @@ When using the default rangeStrategy=auto:
 
 - If a "less than" instruction is found (e.g. `<2`) then `rangeStrategy=widen` will be selected,
 - Otherwise, `rangeStrategy=bump` will be selected.
+
+### Private Modules Authentication
+
+Before running the `cargo` commands to update the `cargo.lock`, Renovate exports `git` [`insteadOf`](https://git-scm.com/docs/git-config#Documentation/git-config.txt-urlltbasegtinsteadOf) directives in environment variables.
+
+The following logic is executed prior to "artifacts" updating:
```

Comment:
```suggestion
Renovate uses this logic before it updates any "artifacts":
```

How about this?

Reply:
I totally agree with you.
I took this sentence in [lib/modules/manager/gomod/readme.md](https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/gomod/readme.md), It could be worth to align this between manager